run/logging-manual/test/system.test.js

```diff
@@ -0,0 +1,114 @@
+const assert = require('assert');
+const {Logging} = require('@google-cloud/logging');
+const request = require('got');
+
+const promisify = require('util').promisify;
+const setTimeoutPromise = promisify(setTimeout);
+
+// Support concurrency by setting the service name to something unique.
+// The service name must be the same used to deploy to Cloud Run.
+const service_name = process.env.CLOUD_RUN_SERVICE_NAME || require('../package').name;
+
+const logging = new Logging({
+  projectId: process.env.GOOGLE_CLOUD_PROJECT
+});
+
+const getLogEntriesPolling = async (filter, max_attempts) => {
+  const WRITE_CONSISTENCY_DELAY_MS = 10000;
+  const MAX_ATTEMPTS = max_attempts || 8;
+  let entries;
+
+  for (i = 0; i < MAX_ATTEMPTS; i++) {
+    await setTimeoutPromise(WRITE_CONSISTENCY_DELAY_MS);
+    entries = await getLogEntries(filter);
+    if (entries[0] && entries[0].length > 0) {
+      return entries[0];
+    }
+  }
+
+  return [];
+};
+
+const getLogEntries = async (filter) => {
+  const preparedFilter = `resource.type="cloud_run_revision" ${filter}`;
+  let entries = await logging.getEntries({
+    filter: preparedFilter,
+    autoPaginate: false,
+    pageSize: 2
+  });
+
+  return entries;
+}
+
+const dateMinutesAgo = (date, min_ago) => {
```

Comment:
Nit: this is probably unnecessary if you use the [`moment`](https://momentjs.com) library

Reply:
Agreed, but this is a fairly simple manipulation and a one-off bit of timestamp math, not worth [2.66 MB](https://packagephobia.now.sh/result?p=moment)

Reply:
I've learned that anything that has to do with dates, is never simple 🤣